stack.yaml

```diff
@@ -0,0 +1,6 @@
+resolver: lts-13.04
```

Comment:
It's probably worth using Nix to build the haskell package, but lets worry about that after we have an implementation working and need to worry about CI and such.   I can do my nix magic once we're done with impl.

Reply:
We could keep both the stack file and the nix setup, as `dhall-haskell` does